frontend/src/components/admin/DocumentUploadsMetabaseDashboard.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import { AuthorizationGuard } from "@/HOC/AuthorizationGuard";
+import * as Permission from "@/constants/permissions";
+
+/**
+ * @class DocumentUploadsMetabaseDashboard displays an iframe housing Document Upload information.
+ */
+
+const jwt = require("jsonwebtoken");
+
+const METABASE_SITE_URL = "https://mds-metabase-empr-mds-prod.pathfinder.gov.bc.ca";
+const METABASE_SECRET_KEY = "623e8cbb9f381e40651494194032c005ba8e92559ba291a8587d1b8e021cb454";
+
+const payload = {
+  resource: { dashboard: 1 },
+  params: {}
+};
+const token = jwt.sign(payload, METABASE_SECRET_KEY);
+
+export class DocumentUploadsMetabaseDashboard extends Component{
```

Comment:
It looks like this doesn't leverage state or context at all. So, it should be able to be converted to a pure function / stateless component